superset/commands/database/update.py

```diff
@@ -47,15 +51,19 @@
 
 
 class UpdateDatabaseCommand(BaseCommand):
+    _model: Optional[Database]
+
     def __init__(self, model_id: int, data: dict[str, Any]):
         self._properties = data.copy()
         self._model_id = model_id
-        self._model: Optional[Database] = None
+        self._model = DatabaseDAO.find_by_id(self._model_id)
```

Comment:
I think our current pattern is still to make all queries/validation on the `run`